Makefile

```diff
@@ -11,22 +11,42 @@ OPAL_POLICY_STORE_URL ?= http://host.docker.internal:8181/v1
 clean:
 	rm -rf *.egg-info build/ dist/
```

Comment:
You should add dist-common, dist-client, dist-server to the clean rule.

Otherwise - every time we do a release - twine might try to re-upload artifacts belonging to old versions.